perllib/FixMyStreet/PhotoStorage.pm

```diff
@@ -0,0 +1,40 @@
+package FixMyStreet::PhotoStorage;
+
+use Moose;
+use Digest::SHA qw(sha1_hex);
+
+has backend => (
+    is => 'ro',
+    lazy => 1,
+    default => sub {
+        my $class = 'FixMyStreet::PhotoStorage::';
+        $class .= FixMyStreet->config('PHOTO_STORAGE_BACKEND') || 'FileSystem';
+        eval "use $class";
```

Comment:
This is fine, but there is a core Module::Load package, that would let you do `load $class;` <shrug>